app/views/layouts/show.html.erb

```diff
@@ -0,0 +1,82 @@
+<div id="chessboard">
+  <div>
+    <div class="white_square" id="KR8"></div>

```

Comment:
It would be great to use `x` and `y` coordinates as identifying values for the cells so it will be really easy to map them to corresponding pieces later on @gvin10

Reply:
I accidentally merged both the branches by accident. I'll create a new pull request to update the html file and application.js file. How should use the x,y coordinates, use them as the id?